frappe/core/doctype/file/file.py

```diff
@@ -734,6 +734,8 @@ def zip_files(files):
 				continue
 			if _file.is_folder:
 				continue
+			if not has_permission(_file, "read"):
+				continue
```

Comment:
should throw instead?

Reply:
I was not sure if this would unnecessarily reveal information

Reply:
let's ensure we're taking strings only. that way we only return what we recieve.

Reply:
No need to throw ig cause in context of desk it shouldn't be possible to add files that aren't permitted (right?)

Reply:
yeah it's ok 👍